README.md

```diff
@@ -16,9 +16,9 @@ This package wraps [HTML5 Canvas](https://developer.mozilla.org/en-US/docs/Web/H
 
 Both Canvas 2D and WebGL are supported.
 
-Both client and server-side scenarios using either Blazor or Razor Components are supported.
+Both WebAssembly (client-side) and server-side scenarios are supported.
```

Comment:
I suggest to use the following as it reflects the current naming which MS announced in the Preview 8 release notes:
Both Blazor Server Apps and Blazor WebAssembly Apps are supported.

Reply:
Good point. Changes made.